PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.h

```diff
@@ -224,6 +225,13 @@ private :
 
 	PluginsManager *_pPluginsManager = nullptr;
 	NppCurrentStatus _nppCurrentStatus;
+
+	LONG _clientWidth;
+	LONG _clientHeight;
+	LONG _initClientWidth;
+	LONG _initClientHeight;
+	RECT _listRect;
+	bool _dialogInitDone;
```

Comment:
What about some initial values for this vars?

Reply:
not sure it worth initialize here because, the init will be done during the WM_INITDIALOG